catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogPropertiesMetadata.java

```diff
@@ -34,6 +36,21 @@ public class HadoopCatalogPropertiesMetadata extends BaseCatalogPropertiesMetada
   // If not, users have to specify the storage location in the Schema or Fileset level.
   public static final String LOCATION = "location";
 
+  /**
+   * The class names of {@link FileSystemProvider} to be added to the catalog. Except built-in
+   * FileSystemProvider like LocalFileSystemProvider and HDFSFileSystemProvider, users can add their
+   * own FileSystemProvider by specifying the class name here. The value can be
+   * 'xxxx.yyy.FileSystemProvider1,xxxx.yyy.FileSystemProvider2'.
+   */
+  public static final String FILESYSTEM_PROVIDERS_CLASSNAMES = "filesystem-providers-classnames";
```

Comment:
The property name here and below is too long, you'd better figure out a better name, I think "filesystem-providers" and "default-filesystem-provider" should be fine.